### PR TITLE
Fix for #6625

### PR DESCRIPTION
**Pull request by kix**
This fix adds a recommendation of installing PHPUnit as a PHAR archive, as suggested in #6625.

### CODE REVIEW

**Comment by javiereguiluz** [comment]
👍

**Comment by alexislefebvre** [comment]
Thanks @kix for taking the time to submit this PR, it looks like the root problem has been fixed: phpspec/prophecy#275

**Comment by alexislefebvre** [comment]
I think that this PR can be closed, PHPUnit can be installed on Symfony 3.1 since the new release of Prophecy this morning.

**Comment by xabbuh** [comment]
Imo it's still a good idea to recommend not to add PHPUnit to the project's requirements.

**Comment by WouterJ** [comment]
I agree with @xabbuh. Thanks for fixing this Stepan!

File: book/testing.rst [hunk 1]
```diff
@@ -17,8 +17,9 @@ it has its own excellent `documentation`_.
 
 .. note::
 
-    It's recommended to use the latest stable PHPUnit version (you will have
-    to use version 4.2 or higher to test the Symfony core code itself).
+    It's recommended to use the latest stable PHPUnit version, `installed as
+    PHAR`_ (you will have to use version 4.2 or higher to test the Symfony core 
+    code itself).
```
Comment (xabbuh) [inline book/testing.rst]: Oh well, now that I look at this again: I think we should remove the note about the core as testing Symfony itself is a bit different (we have our own PHPUnit wrapper script).

Comment (xabbuh) [inline book/testing.rst]: see #6672

**Pull request closed without merging**